src/mame/atari/atarig42.cpp

```diff
@@ -11,10 +11,17 @@
         * Danger Express (1992)
         * Guardians of the 'Hood (1992)
 
-    Known bugs:
-        * ASIC65 for Road Riot not quite perfect
-        * Missing DSPCOM board for Road Riot 4WD
-            +or shared ram pcb that bridges both pcbs for the twin cab kind of like in F1: Exhaust Note and Air Rescue from segas32.cpp
+    Note: Road Riot 4WD sports a communication link with another pcb allowing two players to
+    to compete with each other. The way both pcbs communicate depends on the program version.
+        * The kit version uses an optional com-link board consisting on an ADSP-2105 (10MHz XTAL)
+          routing game link data through the ADSP's serial port.
+        * The dedicated twin cab version uses dual port ram to bridge the two pcbs together (not
+          networked).
```

Comment:
`PCB`, uppercase. Otherwise the PR looks good to me.

Reply:
“RAM” is also an acronym that should be capitalised.

Is the comment actually correct, though?  In particular, is there a reference for it using the ADSP-2105’s serial port?  @Hydreigon223 has been known to make incorrect assumptions about these things.

Saying “not networked” doesn’t make a great deal of sense, either.

Reply:
> “RAM” is also an acronym that should be capitalised.
> 
> Is the comment actually correct, though? In particular, is there a reference for it using the ADSP-2105’s serial port? @Hydreigon223 has been known to make incorrect assumptions about these things.
> 
> Saying “not networked” doesn’t make a great deal of sense, either.

Going to make the capitalization changes in a few minutes. As for the serial port, I'm going to trace the com-link board connector signals to see where they go to. Stay tuned.